Victoradukwu/More-Recipes
# #153824917 set up react environment

#### What does this PR do?
 Install and configure bootstrap and its dependencies for use in application
#### Description of Task to be completed?

- Install bootstrap, popper.js and jquery.js
- add configure webpack.config file for bootstrap, popper an jquery
- import the installed packages as dependencies

#### How should this be manually tested?

- create a bootstrap-formatted html
- start the app
watch the expected css in the output 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#153824917 set up react
#### Screenshots (if appropriate)
N/A


#### Questions:
N/A
